documentation/source/conf.py

```diff
@@ -267,5 +266,24 @@
 # If true, do not generate a @detailmenu in the "Top" node's menu.
 #texinfo_no_detailmenu = False
 
-# For now show the todoy 
+# For now show the todos
 todo_include_todos = True
+
+# -- Extra setup for C documentation with Doxygen and breathe 
+# see also https://breathe.readthedocs.io/en/latest/readthedocs.html
+
+read_the_docs_build = os.environ.get('READTHEDOCS', None) == 'True'
+if read_the_docs_build:
+    subprocess.call('cd ..; doxygen', shell=True)
+    subprocess.call('cd ..; breathe-apidoc -o source/generated source/doxygen/_xml -f', shell=True)
+else:  # local
+    subprocess.call('cd ..; doxygen', shell=True)
+    subprocess.call('cd ..; .venv/bin/breathe-apidoc -o source/generated source/doxygen/_xml -f', shell=True)
```

Comment:
How about adding `.venv/bin` to `$PATH` and  using the same code path for both build types? You can do that by passing `env=` to `subprocess.call`. And if you pass the individual arguments as list you can avoid `shell=True` as well.